va/validation-authority_test.go

```diff
@@ -332,6 +311,54 @@ func TestSimpleHttp(t *testing.T) {
 	test.AssertEquals(t, invalidChall.Error.Type, core.ConnectionProblem)
 }
 
+func TestSimpleHttpRedirectLookup(t *testing.T) {
```

Comment:
Well, neither of those methods should be doing resolution if we're doing things right.  But I take your point that this is only applicable to HTTP at the moment.

We still need to have CNAME/DNAME handling, as that is rather critically important for HTTP and DVSNI.  I don't think this can land until we have that.